service/DevDocs/SendAsyncTimeout.md

```diff
@@ -0,0 +1,36 @@
+# SendAsyncTimeout Requirements
+
+## Overview
+
+DeviceClient class is used by the device client callback delegate to prepare the callback results.
+
+## References
+
+##Exposed API
+
+```csharp
+
+sealed class AmqpServiceClient : ServiceClient
+{
+    ...
+    public async override Task SendAsync(string deviceId, Message message, TimeSpan timeout)
+}
+```
+
+### SendAsyncTimeout Thumbprint
+
+```csharp
+public async override Task SendAsync(string deviceId, Message message, TimeSpan timeout)
```

Comment:
Same as above: should we have a `CancellationToken` instead of `TimeSpan`?

Reply:
The AMQP send message we call already takes a Timespan for timeout. Adding the cancellation token may make it harder than need be.